styles.css

```diff
@@ -0,0 +1,733 @@
+    color: var(--dark-pink);
+}
+.navigation
+{
+    background-color: var(--primary-color);
+    color:rgb(250, 250, 255);
+    height: 4rem;
+    border-radius: 1rem;
+}
+.container
+{
+    padding: 0rem 1rem;
+}
+.navigation .nav-developer
+{
+font-weight: bold;
+padding: 1rem;
+font-size: 25px;
+display: inline;
+}
+.list-non-bullet
+{
+    list-style: none;
+    padding-inline-start: 800px;
+   padding-block-start: 1rem;
+    font-size: 22px;
+    display: inline;
+}
+.list-item-inline
+{
+    display: inline;
+    padding: 0rem 0.5rem
+}
+.navigation .nav-pills
+{
+    text-align: right;
+}
+.navigation .link
+{
+    color: white;
+    text-decoration: none;
+}
+.navigation .link-active
+{
+    font-weight: bold;
+}
+.list-non-bullet-nav2
+
+{
+    list-style: none;
+}
+.nav2{
+    display: flex;
+}
+.nav2-li
+{
+    padding: 1rem;
+    font-size: 20px;
+    color: var(--dark-pink);
+    
+}
+.nav2-li:hover{
+    cursor: pointer;
+    background: rgb(202, 200, 200);
+    border-radius: 5%;
+    width:80px;
+}
+.sec
+{
+    display: flex;
+    flex-direction: row;
+}
+.main-div
+{
+    display: flex;
+}
+.avatar-large
+{
+    width: 90px;
+    border-radius: 60%;
+    padding: 0.5rem;
+}
+.avatar-medium
+{
+    width: 70px;
+    border-radius: 60%;
+    padding: 0.5rem;
+}
+.avatar-small
+{
+    width: 50px;
+    border-radius: 60%;
+    padding: 0.5rem;
+}
+.avatar-extra-small
+{
+    width: 30px;
+    border-radius: 60%;
+    padding: 0.5rem;
+}
+.img-div
+{
+    width: 750px;
+    background-color: black;
+}
+.div-h2
+{
+    color: var(--primary-color);
+    font-size: 28px;
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+}
+
+.sub-div{
+padding: 0.5rem 0rem 0rem 8.5rem;
+
+}
+.div2
+{
+    padding: 0.5rem 0rem 0rem 8.5rem;
+}
+.alert1
+{
+border: solid grey;
+border-radius: 0.7rem;
+font-size: 30px;
+background-color: green;
+width: 60%;
+height: 45px;
+padding: 0.5rem 0rem 0rem 0.5rem;
+color: white;
+position: relative;
+}
+.x-div-alert1
+{
+position: absolute;
+right: 8px;
+top:-2px;
+color: white;
+font-size: 18px;
+font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+}
+.alert2
+{
+border: solid grey;
+border-radius: 0.7rem;
+font-size: 30px;
+background-color: red;
+width: 60%;
+height: 45px;
+padding: 0.5rem 0rem 0rem 0.5rem;
+color: white;
+position: relative;
+}
+.x-div-alert2
+{
+position: absolute;
+right: 8px;
+top:-2px;
+color: white;
+font-size: 18px;
+font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+}
+.alert3
+{
+    border: solid grey;
+border-radius: 0.7rem;
+font-size: 30px;
+background-color: rgb(1, 1, 131);
+width: 60%;
+height: 45px;
+padding: 0.5rem 0rem 0rem 0.5rem;
+color: white;
+position: relative;
+
+}
+.x-div-alert3
+{
+position: absolute;
+right: 8px;
+top:-2px;
+color: white;
+font-size: 18px;
+font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
```

Comment:
Since you're using particular font multiple times. you can maybe create a separate font class and then attach it to the required html so that you don't have to repeat it